Zone integration: update to reflect current SW (#34103)
home-assistant/home-assistant.github.io: source/_integrations/zone.markdown

## Adding a new zone or editing zones

1. Go to {% my zones title="**Settings** > **Areas, labels & zones**" %}.
2. To edit an existing zone, select the edit {% icon "mdi:edit" %} button. To add a new zone, select **Add zone**.
3. Give your zone a name, for example `Nina's office`, or `school`.
4. Pick any icon from [Material Design Icons](https://pictogrammers.com/library/mdi/) and prefix the name with `mdi:`.
   - For example `mdi:school`, `mdi:briefcase`, `mdi:home`, `mdi:cart`, or `mdi:castle`.
   - Note: the icon cannot be changed for the home zone.
5. To change location or radius, under **Edit location**, select edit.
   - To adjust the location, specify the GPS coordinates or drag the icon on the map
   - To change the zone radius, change the size of the zone circle or edit the **Radius** in meters.

  ![Screenshot of the UI for adding or editing a zone](/images/integrations/zone/zone_edit_ui.png)

6. If you want to hide the zone from the frontend and not use the zone for device tracker state, enable **Passive**. You can still use it in automations.
   - **Passive** is not available for the home zone.
7. To save your changes, select **Update**.

> 

Author: no, the order as given is sequential

  - '@home-assistant/core'
ha_domain: zone
ha_integration_type: system
related:
  - docs: /docs/configuration/basic/
    title: Editing basic settings
  - docs: /docs/organizing/areas/
    title: Areas
  - docs: /getting-started/onboarding/
    title: Onboarding
---

Zones allow you to specify certain regions on a map. When a device tracker sees a device to be within a zone, the state will take the name from the zone. Zones can also be used as a [trigger](/docs/automation/trigger#zone-trigger) or [condition](/docs/scripts/conditions/#zone-condition) inside automations. For example, for showing the weather, opening the shades at sunrise, or starting the vacuum when you leave the home.

## About the home zone

> During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it by IP address geolocation.

Reviewer: It uses several processes/methods; depending on what is available, it might even be based on the GPS location provided by the device you are onboarding with.

Suggesting to make it a little less specific, as it is not really relevant here.

```suggestion
During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it automatically.
```

During onboarding, this location was used to create the home zone with a 100&nbsp;m radius. The home zone is a special, pre-defined zone with a few characteristics that set it apart from other zones.

- The name of this zone is defined by the name of your (which defaults to "Home").
- The home zone cannot be deleted and is designated with the home icon in the zone configuration page.
- The home zone's location is used by integrations that are location-based. For example, the [Sun integration](/integrations/sun/), which uses it to calculate the position of the sun relative to your home.
- During onboarding, Home Assistant defined a unit system and time zone based on this location. If you change the location later, unit system and time zone will not be changed automatically anymore.

> - Devices that are in the home zone will not appear on the map in the Home Assistant UI. If you are using your phone for example for presence detection and location based automations, don't be alarm if you don't see your phone on the Map dashboard while you are at home.

Reviewer: ```suggestion
- Devices that are in the home zone will not appear on the map in the Home Assistant UI. If you are using your phone, for example, for presence detection and location based automations, don't be alarmed if you don't see your phone on the Map dashboard while you are at home.
```

## Editing your home zone

> 1. To edit your home zone, go to {% my general title="**Settings** > **System** > **General**" %}.

Reviewer: This can now just be edited under the zones itself. There is no longer a need to handle different steps for the home zone.

Author: well, from what I see, using the current dev version, the handling is a bit different for the home zone. 
- You cannot change the name of the home zone from the Zone page. you need to change the name of the zone. This is why I chose this as the entry point.
- Also, you cannot change the icon of the home zone. 
So, I don't think reusing the same procedure that we use for editing any zone would be entirely accurate.

Reviewer: Honestly, I do not agree. We make the home zone editable via the zones page, the have a more unified and expected experience.

I think is it more valuable to learn the user where zones can be managed in general instead of documenting the weird exception.

Yes, the Home zone has limitations, but those can be described (and should be described anyways)

- The name of this zone is defined by the name of your Home Assistant installation name (which defaults to "Home").
- The home zone cannot be deleted and is designated with the home icon in the zone configuration page.
- The home zone's location is used by integrations that are location-based. For example, the [Sun integration](/integrations/sun/), which uses it to calculate the position of the sun relative to your home.
- During onboarding, Home Assistant defined a unit system and time zone based on this location. If you change the location later, unit system and time zone will not be changed automatically anymore.
- Devices that are in the home zone will not appear on the map in the Home Assistant UI. If you are using your phone, for example, for presence detection and location-based automations, don't be alarmed if you don't see your phone on the Map dashboard while you are at home.

## Adding a new zone or editing zones

1. Go to {% my zones title="**Settings** > **Areas, labels & zones**" %}.
2. To edit an existing zone, select the edit {% icon "mdi:edit" %} button. To add a new zone, select **Add zone**.
3. Give your zone a name, for example `Nina's office`, or `school`.

>    - The home zone always has the name of your home. To change the name for the home, go to {% my general title="**Settings** > **System** > **General**" %}.

Author: ```suggestion
   - The home zone always has the name of your Home Assistant installation. To change the name of the home, go to {% my general title="**Settings** > **System** > **General**" %}.
```

  - '@home-assistant/core'
ha_domain: zone
ha_integration_type: system
related:
  - docs: /docs/configuration/basic/
    title: Editing basic settings
  - docs: /docs/organizing/areas/
    title: Areas
  - docs: /getting-started/onboarding/
    title: Onboarding
---

Zones allow you to specify certain regions on a map. When a device tracker sees a device to be within a zone, the state will take the name from the zone. Zones can also be used as a [trigger](/docs/automation/trigger#zone-trigger) or [condition](/docs/scripts/conditions/#zone-condition) inside automations. For example, for showing the weather, opening the shades at sunrise, or starting the vacuum when you leave the home.

## About the home zone

> During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it automatically.

Reviewer 2: IMHO, this should be combined with the next paragraph. They talk about the same location, and it makes reading/"connecting the dots" easier.

```suggestion
During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it automatically. This location was used to create the home zone with a 100&nbsp;m radius. The home zone is a special, pre-defined zone with a few characteristics that set it apart from other zones.
```

> During onboarding, this location was used to create the home zone with a 100&nbsp;m radius. The home zone is a special, pre-defined zone with a few characteristics that set it apart from other zones.
> 

Reviewer 2: This should be deleted then.
```suggestion
```

> - The name of this zone is defined by the name of your (which defaults to "Home").

Reviewer 2: your.. ?
```suggestion
- The name of this zone is defined by the name of your Home Assistant installation name (which defaults to "Home").
```
Maybe?
